nucypher/characters/lawful.py

```diff
@@ -825,8 +817,8 @@ def __init__(
         operator_address: Optional[ChecksumAddress] = None,
         client_password: Optional[str] = None,
         transacting_power: Optional[TransactingPower] = None,
-        operator_signature_from_metadata=NOT_SIGNED,
-        eth_provider_uri: Optional[str] = None,
+        eth_endpoint: Optional[str] = None,
+        polygon_endpoint: Optional[str] = None,
```

Comment:
👍🏻  These names look great here, it's obvious in retrospect that this is more understandable and accurate.